tests/integration/components/cf-field-test.js

```diff
@@ -3,9 +3,11 @@ import { setupRenderingTest } from "ember-qunit";
 import { render, fillIn } from "@ember/test-helpers";
 import hbs from "htmlbars-inline-precompile";
 import Document from "ember-caluma/lib/document";
+import { setupMirage } from "ember-cli-mirage/test-support";
 
 module("Integration | Component | cf-field", function(hooks) {
   setupRenderingTest(hooks);
+  setupMirage(hooks);
```

Comment:
Why is this needed?

Reply:
https://guides.emberjs.com/v3.10.0/testing/testing-components/#toc_stubbing-services

Reply:
Locate the stub in `/tests/helpers/validator-service-stub.js`